bottlerocket/agents/src/error.rs

```diff
@@ -184,4 +153,8 @@ pub enum Error {
 
     #[snafu(display("The task definition is missing"))]
     TaskDefinitionMissing,
+
+    #[snafu(context(false))]
+    #[snafu(display("{}", source))]
+    Utils { source: agent_utils::Error },
```

Comment:
I can't seem to find any references to this enum variant. Can we remove this?

Reply:
That is a wrapper for any error that happened during a utils call. It is used, but snafu automatically converts it from utils::Error to bottlerocket_agents::Error.

Reply:
I see. `#[snafu(context(false))]` let's us not have to explicitly call `.context(UtilsSnafu)`. That's neat.